test/e2e/play_kube_test.go

```diff
@@ -1004,6 +1042,21 @@ var _ = Describe("Podman play kube", func() {
 		Expect(label).To(ContainSubstring("unconfined_u:system_r:spc_t:s0"))
 	})
 
+	It("podman play kube should share ipc,net,uts when shareProcessNamespace is set", func() {
+		SkipIfRootless("Requires root priviledges for sharing few namespaces")
+		err := writeYaml(sharedNamespacePodYaml, kubeYaml)
+		Expect(err).To(BeNil())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
+		kube.WaitWithDefaultTimeout()
+		Expect(kube.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", "testpod1", "--format", "'{{ .SharedNamespaces }}'"})
+		inspect.WaitWithDefaultTimeout()
+		sharednamespaces := inspect.OutputToString()
+		Expect(sharednamespaces).To(ContainSubstring("ipc"))
```

Comment:
can you add tests for uts and net too, or am I being oblivious and I'm missing them?

Reply:
@TomSweeneyRedHat You are right I missed checking string for `uts` and `net`. Thanks for pointing this out.

Reply:
ah seems like CI flake is still there will open/close PR to re-kick CI.

Reply:
@TomSweeneyRedHat this is resolved.

Reply:
also need `pid` namespace

Reply:
@zhangguanzhang Bug was with only `ipc, net, uts` but sure case should also pass with `pid`

Reply:
@zhangguanzhang @TomSweeneyRedHat Added `pid` as well let me know if anything else needs to be done.

I think `rdoproject.org/github-check` is a flake , repush should fix it. But let me know if this check is mandatory.

Reply:
@zhangguanzhang ah it was just a flake force pushed made checks happy again.